groovy/postInit/chemistry/organic_chemistry/polymers/PolyvinylChain.groovy

```diff
@@ -26,41 +28,12 @@ TBR.recipeBuilder()
     .EUt(30)
     .buildAndRegister()
 
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('chlorine') * 100)
-    .fluidInputs(fluid('ethylene') * 50)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('hydrogen_chloride') * 50)
-    .duration(5)
-    .EUt(30)
-    .buildAndRegister()
-
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethylene') * 50)
-    .fluidInputs(fluid('hydrogen_chloride') * 50)
-    .fluidInputs(fluid('oxygen') * 50)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('water') * 50)
-    .duration(10)
-    .EUt(30)
-    .buildAndRegister()
-
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethane') * 50)
-    .fluidInputs(fluid('chlorine') * 200)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('hydrogen_chloride') * 150)
-    .duration(10)
-    .EUt(30)
-    .buildAndRegister()
-
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethane') * 50)
-    .fluidInputs(fluid('hydrogen_chloride') * 50)
-    .fluidInputs(fluid('oxygen') * 100)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('water') * 100)
-    .duration(10)
+// Dehydrochlorination
+PYROLYSE_OVEN.recipeBuilder()
+    .fluidInputs(fluid('dichloroethane') * 1000)
+    .fluidOutputs(fluid('vinyl_chloride') * 1000)
+    .fluidOutputs(fluid('hydrogen_chloride') * 1000)
+    .duration(100)
     .EUt(30)
```

Comment:
this should produce a mix of chlorinated hydrocarbon byproducts:
![image](https://github.com/user-attachments/assets/3fb24685-9094-4a17-8fae-88398026970c)

Reply:
Hmm, I see
![image](https://github.com/user-attachments/assets/c841a936-539a-44c0-b327-8e5422f6e3e7)
![image](https://github.com/user-attachments/assets/375d89a6-98c3-49c0-b824-447ee98024c1)
Would be fun to do, although, wouldn't it be too much CBT for MV player?

Reply:
just use distillation tower to distill it
pre-HV players will have to accept the costs of losing byproducts.